pyalink/adult.ipynb

```diff
@@ -107,7 +107,7 @@
     "        .setPredictionCol(\"pred\").setPredictionDetailCol(\"detail\")\n",
     "model = pipeline.add(logistic).fit(adult_batch)\n",
     "\n",
-    "predictBatch = model.transform(adult_batch)\n",
+    "predictBatch = model.transform(adult_stream)\n",
```

Comment:
Thanks @xixici 

The model is used to predict the samples in batch mode and to eval in batch mode rather than the stream mode